packages/askui-nodejs/src/execution/dsl.ts

```diff
@@ -3713,7 +3713,17 @@ export class FluentFiltersOrRelationsGetter extends FluentFiltersGetter {
 // Commands
 export abstract class Getter extends FluentCommand {
   /**
-   * Returns the filtered element
+   * Returns an array with all filtered elements.
+   * A detected element has the following properties:
+   * - `name` of the element
+   * - `text` content of element
+   * - `colors` of element
+   * - `bndbox`: location of element described with coordinates of a bounding box
+   * **Examples:**
+   * ```typescript
+   * const icon = await aui.get().icon().withText('care left').exec();
```

Comment:
Can you add an example of what is returned ;)

```suggestion
   * const icon = await aui.get().icon().withText('care left').exec();
   * console.log(icon)
   * // [{ name: 'icon', text: '...', colors: [...], ... }, ...]
```

(add the details to what is logged, maybe a multiline comment would be better)